pkgs/by-name/fi/firefly-iii-data-importer/package.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, unzip }:
+let
+  version = "1.5.2";
+in
+stdenv.mkDerivation rec {
+  pname = "firefly-iii-data-importer";
+  inherit version;
+
+  src = builtins.fetchurl {
+    url = "https://github.com/firefly-iii/data-importer/releases/download/v${version}/DataImporter-v${version}.zip";
+    sha256 = "sha256:1ssxhgd6x4lp8ak5zkgj02hvpvdsgxafli9syb9lwrkkr9z8lyyg";
+  };
+
+  buildInputs = [ unzip ];
+
+  phases = [
+    "unpackPhase"
+    "patchPhase"
+    "installPhase"
+  ];
+
+  unpackPhase = ''
+    unzip $src
+  '';
+
+  postPatch = ''
+    # Get the storage directory from the env variable instead of in the nix store
+    sed -i "s/app()->storagePath/env('FF3_DATA_IMPORTER_STORAGE') . '\/' . \$path; #/g" \
+      vendor/laravel/framework/src/Illuminate/Foundation/helpers.php
+  '';
+
+  installPhase = ''
+    cp -r . $out
+  '';
+}
```

Comment:
You should add a meta section in the package as well. Again, refer to the firefly-iii package.